src/test/java/com/jcabi/aspects/aj/MnemosTest.java

```diff
@@ -86,4 +91,126 @@ public String toString() {
         );
     }
 
+    /**
+     * Mnemos can build a string from an int array.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void buildsTextFromIntArray() throws Exception {
+        final Object[][] pairs = new Object[][] {
+            new Object[] {new int[0], EMPTY_ARRAY},
+            new Object[] {new int[] {1}, "[1]"},
+            new Object[] {new int[] {1, 2, 3}, "[1, 2, 3]"},
+        };
+        for (final Object[] pair : pairs) {
```

Comment:
this for is exactly the same in all the array tests, let's move it to a private method